test/webhook_test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+sed -ri "s|'(quay.io/reactiveops/polaris:).+'|'\1${CIRCLE_SHA1}'|" ./deploy/webhook.yaml
+kubectl apply &> /dev/null -f ./deploy/webhook.yaml
+sleep 15
```

Comment:
If we can avoid it, I always try to seek a more deterministic approach to asynchronous tasks. In this scenario we're introducing a race condition that's not an actual test. If we aren't ready after 15 seconds, is this an important bit of knowledge for test cases? 

Another approach may be to sleep 15, and write a "check condition" to see if we're in the desired state after that 15 seconds. Another approach (if the "read in 15 seconds" is arbitrary and not a real test) is to just write that test and loop until it passes or timeout after X second so we're not forever stuck checking that the webhook initialized correctly.

Since you have a "known bad config", we can probably use that in a while loop to proceed when the webhook successfully blocks the ability to apply the known bad config.

Happy to discuss more of what I mean here :)

Reply:
Nick makes a good point. Here's a safer way to wait for the webhook:
```bash
timeout=15
while ! kubectl get pods -n polaris | grep "polaris-webhook.*Running"; do
  echo "Waiting for webhook to start..."
  if [ $timeout -eq 0 ]; then
    echo "Timed out while waiting for webhook to start"
    exit 1
  fi
  timeout=$((timeout-1))
  sleep 1
done
echo "Webhook started!"
```